jamf_protect/assets/dashboards/jamf_protect_overview.json

```diff
@@ -0,0 +1,1400 @@
+{
+  "title": "Jamf Protect Overview",
+  "description": "This is an overview of your organization's Jamf Protect telemetry, unified logs and alerts. From here you can discover insights on the system, user, application and network activity originating from your endpoints.\n",
+  "widgets": [
+    {
+      "id": 3411659769839050,
+      "definition": {
+        "title": "New group",
+        "banner_img": "/static/images/integration_dashboard/jamf-protect_hero_1.png",
+        "show_title": false,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 4567712873518980,
+            "definition": {
+              "type": "note",
+              "content": "Use this dashboard to monitor your Jamf Protect telemetry, unified logs and alerts, giving you greater visibility into security-related events generated from your Jamf deployment.\n\nOnce you’ve enabled the Jamf Protect integration in your Datadog account, you’ll be able to easily search, filter, and alert on all Jamf protect events in Datadog.\n",
+              "background_color": "transparent",
+              "font_size": "16",
+              "text_align": "left",
+              "vertical_align": "top",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": { "x": 0, "y": 0, "width": 6, "height": 4 }
+          }
+        ]
+      },
+      "layout": { "x": 0, "y": 0, "width": 6, "height": 7 }
+    },
+    {
+      "id": 6238873771110860,
+      "definition": {
+        "title": "Jamf Protect Log Overview",
```

Comment:
Should we specify "Security Logs"?

Reply:
Jamf protect is a security feature/product, so I think it would be assumed that they are security based logs